packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -503,7 +567,10 @@ export class DeltaManager
             pendingOps: this.pending.length, // Do we have any pending ops?
             pendingFirst: pendingSorted[0]?.sequenceNumber, // is the first pending op the one that we are missing?
             haveHandler: this.handler !== undefined, // do we have handler installed?
-            closed: this.closed,
+            inboundLength: this.inbound.length,
+            inboundPaused: this.inbound.paused,
+            inboundMaxCallers: stacks.length,
+            inboundPauser: stacks.join("\n\n"),
```

Comment:
Would be nice if we can measure the amount of time we paused for the duration we measure connection issues.

Reply:
Logged https://github.com/microsoft/FluidFramework/issues/8214